lib/driver.js

```diff
@@ -143,7 +143,10 @@ class AndroidDriver extends BaseDriver {
       await this.startAndroidSession(this.opts);
       return [sessionId, this.caps];
     } catch (e) {
-      await this.deleteSession();
+      try {
+        await this.deleteSession();
+      } catch (ignored) {
+      }
```

Comment:
Can you add a comment here describing the reason for this construct? It is non-obvious from the code, and will be so when others look at it later, outside the context of this PR.